cl_manycore/regression/library/test_manycore_init.h

```diff
@@ -1,11 +1,9 @@
 #pragma once
 #ifdef COSIM
```

Comment:
How did these #ifdef cosims sneak back in? I thought I removed all of them

Reply:
Got them!